test/forms/labels/input-label.test.js

```diff
@@ -38,3 +38,13 @@ test('when tooltip provided - toggle tooltip', () => {
   wrapper.find('span.tooltip-trigger').simulate('click')
   expect(wrapper.find('div.tooltip-content.is-active').exists()).toEqual(false)
 })
+
+test('when required true - no custom indicator provided', () => {
```

Comment:
Can you put the expected result in the name of the test? Eg. `when no custom required indicator provided, do not show required indicator`